Mathlib/Order/SuccPred/Basic.lean

```diff
@@ -424,6 +451,32 @@ section NoMaxOrder
 
 variable [NoMaxOrder α]
 
+@[simp]
+theorem lt_succ_iff : a < succ b ↔ a ≤ b :=
+  lt_succ_iff_of_not_isMax <| not_isMax b
+
+theorem succ_le_succ_iff : succ a ≤ succ b ↔ a ≤ b := by simp
+
+theorem succ_lt_succ_iff : succ a < succ b ↔ a < b := by simp
+
+alias ⟨le_of_succ_le_succ, _⟩ := succ_le_succ_iff
+
+alias ⟨lt_of_succ_lt_succ, _⟩ := succ_lt_succ_iff
```

Comment:
In general, we like to glue together series of similar lemmas. It's a visual cue that they are related
```suggestion
theorem succ_lt_succ_iff : succ a < succ b ↔ a < b := by simp

alias ⟨le_of_succ_le_succ, _⟩ := succ_le_succ_iff
alias ⟨lt_of_succ_lt_succ, _⟩ := succ_lt_succ_iff
```